packages/app/src/modules/options/config.js

```diff
@@ -25,33 +25,33 @@ import singleValueConfig from './singleValueConfig'
 import barConfig from './barConfig'
 import yearOverYearConfig from './yearOverYearConfig'
 
-export const getOptionsByType = type => {
+export const getOptionsByType = (type, hasCustomAxes) => {
     switch (type) {
         case VIS_TYPE_COLUMN:
-            return columnConfig
+            return columnConfig(hasCustomAxes)
```

Comment:
This potentially needs to be expanded into an object with various flags...

Reply:
Yes indeed, if more props are needed then this should be changed to an object instead. Hopefully `hasCustomAxes` (which is the same as "has multiple axes") will suffice for the near future though. I think you can use it for hiding the color sets as well, right?

Reply:
Yes, for that it will work.
I'm thinking about the "Legends applied to bar and column charts only one data item should be allowed in the series with only one axis" type of condition 😆